t/test_paths.pm

```diff
@@ -34,12 +39,13 @@ BEGIN {
     $pf::file_paths::firewall_sso_config_file = catfile($test_dir,'data/firewall_sso.conf');
     $pf::file_paths::config_file = catfile($test_dir,'data/pf.conf');
     $pf::file_paths::pf_config_file = catfile($test_dir,'data/pf.conf');
```

Comment:
Only thing is that the content of pf.conf should be predictable.

For example, in mine I have proxy passthroughs that make the unit tests fail.

I won't have a choice to point it to mine since it contains my IP addresses.

Would it sound right to have the pf.conf overriden via ENV and have the IPs and DB options overriden via ENV also ?

This way you don't have to absolutly override that value and do it if you want to while keeping the other ones so setup dependant variables can be set.